tests/test_core.py

```diff
@@ -7740,15 +7741,26 @@ def test_minimal_runtime_hello_world(self, args):
     'fs': (['-s', 'FORCE_FILESYSTEM'],),
     'nofs': (['-s', 'NO_FILESYSTEM'],),
```

Comment:
Wasn't this changed to
```
    'fs': ('FORCE_FILESYSTEM',),
    'nofs': ('NO_FILESYSTEM',),
```
in #13186? But that PR was merged into not the main branch but this branch, but this branch doesn't have that...

Reply:
Oh darn I didn't mean to marge #13186 yet.

The idea was to land this change first.   If you lg this I will re-upload #13186.